users/views.py

```diff
@@ -1,14 +1,18 @@
 import json
 import re
 from json.decoder import JSONDecodeError
+import bcrypt
+import jwt
```

Comment:
```suggestion
import json
import re
import bcrypt
import jwt
from json.decoder import JSONDecodeError
```
이게 조금 더 가독성이 좋을 것 같습니다.

Reply:
수정했습니다